### PR TITLE
Sample pull request

### LINKED ISSUE #1: Create 3 separate folders for ML, Backend, and frontend.

### PR DESCRIPTION
**Pull request by riyakb**
Have created separate folders for ML, Backend, Frontend.
Also have created Data folder, and added schema and generation scripts to it.

### CODE REVIEW

File: Backend/README [hunk 1]
```diff
@@ -0,0 +1 @@
+This folder contains all the backend related files
```
Comment (nityendrasingh) [inline Backend/README]: use README.md

File: Frontend/README [hunk 2]
```diff
@@ -0,0 +1 @@
+This folder contains all the frontend related files
```
Comment (nityendrasingh) [inline Frontend/README]: README.md

File: ML/README [hunk 3]
```diff
@@ -0,0 +1 @@
+This folder contains all the ML related files
```
Comment (nityendrasingh) [inline ML/README]: README.md

File: Backend/README (hunk 1 above)
Comment (riyakb) [reply Backend/README]: Done!

**Review by nityendrasingh — APPROVED** [review]
Thanks!

**Pull request merged**